src/_gettsim/transfers/kindergeld.py

```diff
@@ -13,12 +15,17 @@
         "source_col": "kindergeld_anspruch",
         "aggr": "sum",
     },
+    "kindergeld_übertrag_m": {
```

Comment:
The additions in this module do not really fit here. How about a new module in the `arbeitsl_geld_2` directory?

Reply:
Good idea!